File: src/api/queries/aggregate/AggregateQuerySession.cs
using System;
using System.Collections.Generic;
using DVAN.Accessibility;
using DVAN.Population;
using DVAN.Routing;
using NetTopologySuite.Geometries;

namespace DVAN.API
{
    class AggregateQuerySession
    {
        public Guid id { get; set; }

        public IPopulationView population_view { get; set; }

        public ICatchment catchment { get; set; }
    }
}

File: src/main/java/org/tud/oas/api/queries/aggregate/AggregateQuerySession.java
package org.tud.oas.api.queries.aggregate;

import java.util.UUID;

import org.tud.oas.demand.IDemandView;
import org.tud.oas.routing.ICatchment;

class AggregateQuerySession {
    public UUID id;

    public IDemandView demand_view;

    public ICatchment catchment;

    public AggregateQuerySession(UUID id, IDemandView population_view, ICatchment catchment) {
        this.id = id;
        this.demand_view = population_view;
        this.catchment = catchment;
    }
}